src/openrct2/Game.cpp

```diff
@@ -127,17 +127,21 @@ GAME_COMMAND_CALLBACK_POINTER * game_command_callback_get_callback(uint32_t inde
 
 void game_increase_game_speed()
 {
-    gGameSpeed = std::min(gConfigGeneral.debugging_tools ? 5 : 4, gGameSpeed + 1);
-    if (gGameSpeed == 5)
-        gGameSpeed = 8;
+    if (gGameSpeed < (gConfigGeneral.debugging_tools ? GameSpeedHyper : GameSpeedTurbo))
+        gGameSpeed++;
+
+    if ( game_is_paused() )
```

Comment:
no spaced inside the parentheses. Use clang-format to format correctly automatically.